bookwyrm/signatures.py

```diff
@@ -15,7 +15,7 @@
 def create_key_pair():
     """a new public/private key pair, used for creating new users"""
     random_generator = Random.new().read
-    key = RSA.generate(1024, random_generator)
+    key = RSA.generate(2048, random_generator)
     private_key = key.export_key().decode("utf8")
     public_key = key.publickey().export_key().decode("utf8")
```

Comment:
With version bum of pycryptodome to 3.9.4 key.publickey needs to change to key.public_key.

Reply:
I think merge of main branch will do this.